robot/client/client.go

```diff
@@ -620,22 +621,29 @@ func (rc *RobotClient) resources(ctx context.Context) ([]resource.Name, []resour
 		defer cancel()
 	}
 
-	resp, err := rc.client.ResourceNames(ctx, &pb.ResourceNamesRequest{})
```

Comment:
Wait, are you changing robots to use machine status instead of resource names to populate remote resources?